src/modules/sensors/CMakeLists.txt

```diff
@@ -35,9 +35,8 @@ px4_add_module(
 	MODULE modules__sensors
 	MAIN sensors
 	PRIORITY "SCHED_PRIORITY_MAX-5"
-	STACK_MAIN 1200
+	STACK_MAIN 2000
```

Comment:
This shouldn't have made it in. When stackcheck is enabled sensors is somehow causing a hard fault when the main exits. I tried throwing a ton of stack at it.